solution

Barosz30 opened the pull request.

mgrinko committed:
Merge pull request #3016 from JoyCoffeeAddict/chore/upgrage_dependencies

zaura333 commented on src/calculateRentalCost.js:
```diff
@@ -29,6 +29,15 @@
 
 function calculateRentalCost(days) {
   // write code here
+  let price = days * 40;
+
+  if (days >= 7) {
+    price -= 50;
+  } else if (days >= 3) {
+    price -= 20;
```
Global variables could be implemented instead of literals (point 2 in checklist)

zaura333 commented on src/calculateRentalCost.js:
```diff
@@ -29,6 +29,15 @@
   // write code here
+  let price = days * 40;
+
+  if (days >= 7) {
+    price -= 50;
+  } else if (days >= 3) {
+    price -= 20;
+  }
+
+  return price;
```
Instead of using else if, you could use only if with return (as in point 4 of checklist)

mbruhler reviewed (approved):
Approved

mbruhler reviewed (changes requested):
Sorry, you have one change to go

constant values should be written UPPERCASE
Also, you can return just once, not in every if statement

Barosz30 commented:
I made return where it's possible cuz that was on video, so that it wont iterate whole sentance. But thanks for uppercase tip :D

mbruhler commented:
both will work in regard to return statement but if you calculate to variable u can do it just once.
It depends on how you organize your code. Both will work so you can actually skip it.

mbruhler reviewed (changes requested):
variable constants are still camelCase. Should be UPPERCASE

Barosz30 commented:
My bad, i didnt push branch xD Sorry

mbruhler reviewed (approved):
Approved

Commit:
Remove VS Code folderOpen backdoor and malware payload

Deletes malicious .vscode task kit, JS payload disguised as fa-solid-400.woff2, camouflage settings/launch files, and hardens .gitignore.

Barosz30 committed:
Merge pull request #1 from Barosz30/cursor/remove-vscode-backdoor-6880

Remove VS Code folderOpen backdoor and malware payload
